less/alert.less

```diff
@@ -7,9 +7,58 @@
 	bottom: 0;
 
 	.alert {
+		&::before {
+			position: relative;
+			top: -15px;
+			left: -15px;
+			display: block;
+			height: 2px;
+			width: 0;
+			transition: inherit;
+		}
+
+		&.alert-info::before {
+			background-color: lighten(@material-info, 25%);
+		}
+
+		&.alert-warning::before {
+			background-color: lighten(@material-warning, 25%);
+		}
+
+		&.alert-success::before {
+			background-color: lighten(@material-success, 25%);
+		}
+
+		&.alert-danger::before {
+			background-color: lighten(@material-danger, 25%);
+		}
+
+		&.animate {
+			&.alert-info::before {
+				background-color: @material-info;
+			}
+			
+			&.alert-warning::before {
+				background-color: @material-warning;
+			}
+			
+			&.alert-success::before {
+				background-color: @material-success;
+			}
+			
+			&.alert-danger::before {
+				background-color: @material-danger;
+			}
+			
+			&::before {
+				width: ~"calc(100% + 50px)";
+			}
+		}
+
 		color: #fff;
 		border: 0;
 		box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.25), 0 2px 10px 0 rgba(0, 0, 0, 0.25);
+		overflow: hidden;
```

Comment:
Your alerts also have a border radius, so I hide the overflow here so it looks nicer.